cmd/mimir/config-descriptor.json

```diff
@@ -727,6 +727,49 @@
           "fieldValue": null,
           "fieldDefaultValue": null
         },
+        {
+          "kind": "block",
+          "name": "retry_after_header",
+          "required": false,
+          "desc": "",
+          "blockEntries": [
+            {
+              "kind": "field",
+              "name": "enabled",
+              "required": false,
+              "desc": "Enabled controls inclusion of the Retry-After header in the response: true includes it for client retry guidance, false omits it.",
+              "fieldValue": null,
+              "fieldDefaultValue": false,
+              "fieldFlag": "distributor.retry-after-header.enabled",
+              "fieldType": "boolean",
+              "fieldCategory": "experimental"
+            },
+            {
+              "kind": "field",
+              "name": "base_seconds",
+              "required": false,
+              "desc": "Base duration in seconds for calculating the Retry-After header in responses to 429/5xx errors.",
```

Comment:
It might be good to mention that this is a "Base exponential backoff duration"... here, and elsewhere that it's documented.

Reply:
Reason is that value of Retry-After header is expected to be integer number of seconds. (Or absolute date with seconds resolution, but we don't use that format). [Link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After)